### PR TITLE
Improve validation buttons storing methods as attributes

### PR DESCRIPTION
**Pull request by n8kim1**
validateReaction didn't work, because one of the button's onclick methods was missing a parenthesis. Dang...

Also, onclick methods had been stored as custom `button-onclick` attributes in the HTML, and dynamically turned into `onclick` attributes when the editor loaded. This led to my IDE interpreting the attribute method as a generic block of text, not JS, and so it never noticed the error. Will change the code to represent JS as real JS.

### CODE REVIEW

File: editor/js/reaction.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ async function init(fileName, index) {
   loadReaction(reaction);
   clean();
   // Trigger reaction-level validation.
-  setTimeout(validateReaction, 1000);
```
Comment (skearnes) [inline editor/js/reaction.js:68]: Why is this removed? I thought there was some startup lag that this was intended to control for?

Comment (n8kim1) [reply editor/js/reaction.js:68]: Didn't solve the problem we were originally trying to solve... I figured the timeout wasn't necessary. 

After further thought, even though I can't notice any issues in my own testing, I think the timeout is still safer (perhaps if startup takes a while for example). I'll put it back.

Comment (n8kim1) [reply editor/js/reaction.js:68]: After even _more_ testing, validateReaction should be okay without a timeout. Loading, adding, and starting to show the validation divs all take place before validateReaction is called. Even though we have a "show" transition, the divs are still visible before the transition actually finishes.

I'll stick with no timeout for now...

**Comment by n8kim1** [comment]
@skearnes added a little bit more, could you take another look? Sorry I left it hanging.

**Comment by skearnes** [comment]
Do you need anything else here? I see that the PR is still marked as a draft.

**Comment by n8kim1** [comment]
Nope, just rebase and merge that I'm finally having a chance now -- thanks for the check!
